### DISCUSSION TITLE
Help me out I am deploying my site on the railway rather than vercel but still server issues

### DISCUSSION THREAD

**Discussion by siddharth-shah-17** [sahilatahar/Code-Sync#17] [Q&A]
Help me out I am deploying my site on the railway rather than vercel but still server issues
It still doesn't connect with the server and says that blocking of API from railway and the url.railway.app/socketio/---- doesn't exist on project
I am also facing this issue
@sahilatahar

**Comment by sahilatahar** [comment]
Does railway support socket connection? check it.
